sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/AzureMonitorExporterOptions.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace OpenTelemetry.Exporter.AzureMonitor
+{
+    public class AzureMonitorExporterOptions : ClientOptions
+    {
+        private const int DefaultCapacityKiloBytes = 50 * 1024;
+        private const long StorageCapacity = DefaultCapacityKiloBytes * 1024;
+
+        public string ConnectionString { get; set; }
```

Comment:
Is the connection string optional?

Reply:
No. We extract IKey from connection string. I defined all the properties, need to add logic for all the properties in next PRs.